plugins/woocommerce/includes/class-wc-session-handler.php

```diff
@@ -449,19 +553,6 @@ public function forget_session() {
 		$this->_has_cookie  = false;
 	}
 
-	/**
-	 * When a user is logged out, ensure they have a unique nonce by using the customer/session ID.
-	 *
-	 * @deprecated 5.3.0
-	 * @param int $uid User ID.
-	 * @return int|string
-	 */
-	public function nonce_user_logged_out( $uid ) {
```

Comment:
Couldn't find public usage of this function, but there's a hook with a similar name,

Reply:
It was deprecated 4 years ago and replaced. Used to be hooked in.